birdhouse/optional-components/x-robots-tag-header/.gitignore

```diff
@@ -0,0 +1 @@
+config/proxy/conf.extra-service.d/x-robots-tag-header.conf
```

Comment:
You'll need to keep both the old and new ignore name because it'll break autodeploy for servers that already deployed the old instanciated template filename.

It'll have to be a new entry to the gitignore at optional-components because you also renamed the directory name.